content/reference/admin/private_locations/installation/azure/index.md

```diff
@@ -49,7 +49,7 @@ Write down the storage account name, you will need it in further steps.
 
 Once again, refer to the [Azure File Share documentation](https://learn.microsoft.com/en-us/azure/storage/files/storage-how-to-use-files-portal?tabs=azure-portal) for details.
 
-In your storage account, let's create a File Share for the control plane configuration file:
+In your storage account, let's create a File Share for the [Control Plane configuration]({{< ref "../../configuration" >}}) file:
 * Access to your storage account page
```

Comment:
If this is a procedure that should be followed sequentially it is better to use a numbered list than an unordered list.